corehq/motech/repeaters/models.py

```diff
@@ -198,6 +203,7 @@ def all_ready(self):
 
 class SQLRepeater(models.Model):
```

Comment:
should this model be `abstract` https://docs.djangoproject.com/en/3.2/topics/db/models/#abstract-base-classes
If no records will be in this table, and instead only in the inherited models, that would enforce the constraint.

Reply:
Will multi table inheritance allow us to query all subclasses at once? if so I agree that is much better than an abstract class, as it would allow `Repeater.objects.filter(domain='x')` rather than 
```
for rc in repeater_classes:
    rc.objects.filter(domain='x')
```
If so, that definitely sounds correct. If not we may want to look into making these a single table instead, because that kind of query seems pretty common (every time a case or form changes) and we have a ton of repeater classes.